openstack/workflow/v2/crontriggers/requests.go

```diff
@@ -78,3 +79,54 @@ func Get(client *gophercloud.ServiceClient, id string) (r GetResult) {
 	_, r.Err = client.Get(getURL(client, id), &r.Body, nil)
 	return
 }
+
+// ListOptsBuilder allows extension to add additional parameters to the List request.
+type ListOptsBuilder interface {
+	ToCronTriggerListQuery() (string, error)
+}
+
+// ListOpts filters the result returned by the List() function.
+type ListOpts struct {
```

Comment:
I think there are some fields missing here?

https://github.com/openstack/mistral/blob/master/mistral/api/controllers/v2/cron_trigger.py#L106-L112

In addition, I think `SortKey` should be `SortKeys` and `SortDir` should be `SortDirs`?